AsyncDisplayKit/ASPagerNode.m

```diff
@@ -0,0 +1,74 @@
+//
+//  ASPagerNode.m
+//  AsyncDisplayKit
+//
+//  Created by Levi McCallum on 12/7/15.
+//  Copyright © 2015 Facebook. All rights reserved.
+//
+
+#import "ASPagerNode.h"
+
+#import <AsyncDisplayKit/AsyncDisplayKit.h>
+
+@interface ASPagerNode () <ASCollectionViewDataSource, ASCollectionViewDelegateFlowLayout> {
+  UICollectionViewFlowLayout *_flowLayout;
+}
+
+@end
+
+@implementation ASPagerNode
+
+- (instancetype)init
+{
+  _flowLayout = [[UICollectionViewFlowLayout alloc] init];
+  _flowLayout.scrollDirection = UICollectionViewScrollDirectionHorizontal;
+  _flowLayout.minimumInteritemSpacing = 0;
+  _flowLayout.minimumLineSpacing = 0;
+  
+  self = [super initWithCollectionViewLayout:_flowLayout];
+  if (self != nil) {
+    self.view.asyncDataSource = self;
+    self.view.asyncDelegate = self;
+    
+    self.view.pagingEnabled = YES;
+    self.view.allowsSelection = NO;
+    self.view.showsVerticalScrollIndicator = NO;
+    self.view.showsHorizontalScrollIndicator = NO;
+    
+    ASRangeTuningParameters tuningParams = { .leadingBufferScreenfuls = 1.0, .trailingBufferScreenfuls = 1.0 };
+    [self setTuningParameters:tuningParams forRangeType:ASLayoutRangeTypePreload];
+    [self setTuningParameters:tuningParams forRangeType:ASLayoutRangeTypeRender];
+  }
+  return self;
+}
+
+- (void)reloadData
+{
+  [self.view reloadData];
+}
+
+- (void)setTuningParameters:(ASRangeTuningParameters)tuningParameters forRangeType:(ASLayoutRangeType)rangeType
```

Comment:
It's not obvious that this will trigger view creation.  Suggest putting it on ASCollectionNode - and creating a new data type similar to ASPendingViewState that both ASCollectionNode and/or ASTableNode (possibly different data types for each) can use to store properties like the delegate / dataSource / tuningParameters before the view is created, and apply them in -didLoad.